pkg/expression/column.go

```diff
@@ -442,6 +447,48 @@ func (col *Column) GetStaticType() *types.FieldType {
 	return col.RetType
 }
 
+// Hash64 implements HashEquals.<0th> interface.
+func (col *Column) Hash64(h base.Hasher) {
+	col.RetType.Hash64(h)
+	h.HashInt64(col.ID)
+	h.HashInt64(col.UniqueID)
+	h.HashInt(col.Index)
+	//col.VirtualExpr.Hash64(h)
+	h.HashString(col.OrigName)
+	h.HashBool(col.IsHidden)
+	h.HashBool(col.IsPrefix)
+	h.HashBool(col.InOperand)
+	col.collationInfo.Hash64(h)
+	h.HashInt64(col.CorrelatedColUniqueID)
+}
+
+// Equals implements HashEquals.<1st> interface.
+func (col *Column) Equals(other any) bool {
+	if other == nil {
+		return false
+	}
+	var col2 *Column
+	switch x := other.(type) {
+	case Column:
+		col2 = &x
+	case *Column:
+		col2 = x
+	default:
+		return false
+	}
+	return col.RetType.Equal(col2.RetType) &&
+		col.ID == col2.ID &&
+		col.UniqueID == col2.UniqueID &&
+		col.Index == col2.Index &&
+		//col.VirtualExpr.Equals(col2.VirtualExpr) &&
```

Comment:
Is it to keep or remove?

Reply:
VirtualExpr is scalar expression，which should introduce HashEquals interface, which i haven't done yet

Reply:
that's why i just comment it for later opening